pkg/clusterresource/vsphere.go

```diff
@@ -46,9 +46,6 @@ type VSphereCloudBuilder struct {
 	// IngressVIP is the virtual IP address for ingress
 	IngressVIP string
 
-	// DNSVIP is the virtual IP address for DNS
-	DNSVIP string
```

Comment:
A little strange, is this meant to be in this PR?

Reply:
Yes. The `DNSVIP` was removed from the installer types in the version of the installer that I am vendoring.